tests/worker/test_workflow.py

```diff
@@ -5276,6 +5276,12 @@ async def test_workflow_current_update(client: Client, env: WorkflowEnvironment)
         )
 
 
+def skip_unfinished_handler_tests_in_older_python():
+    # These tests reliably fail or timeout in 3.9
+    if sys.version_info < (3, 10):
+        pytest.skip("Skipping unfinished handler tests in Python < 3.10")
```

Comment:
Yeah, I still haven't pinned this down. I think it's mostly or always under `time-skipping`? This PR shows that in fact they do not seem to reliably fail any longer: https://github.com/temporalio/sdk-python/pull/724 Basically I was trying to study the problem and of course they stopped failing.